desktop/apps/gene/templates/meta.jade

```diff
@@ -9,5 +9,4 @@ meta( property="og:image", content=gene.imageUrl('big_and_tall') )
 meta( property="og:type", content=(sd.FACEBOOK_APP_NAMESPACE + ":gene" ) )
 meta( property="twitter:card", content="summary" )
 if sd.INCLUDE_ESCAPED_FRAGMENT
-  if includeMetaFragment
```

Comment:
As far as I could tell, this boolean appears nowhere and is never set or used.

Reply:
Yeah, that looks like an odd copy paste fail—the `sd. INCLUDE_ESCAPED_FRAGMENT` above is what we use for this.